controllers/vspherevm_controller.go

```diff
@@ -462,7 +463,7 @@ func (r vmReconciler) isWaitingForStaticIPAllocation(vmCtx *capvcontext.VMContex
 	return false
 }
 
-func (r vmReconciler) reconcileNetwork(vmCtx *capvcontext.VMContext, vm infrav1.VirtualMachine) {
+func (r vmReconciler) reconcileNetwork(_ context.Context, vmCtx *capvcontext.VMContext, vm infrav1.VirtualMachine) {
```

Comment:
Hi @Ankitasw @zhanggbj , I put a new context parameter here is because I am wondering if we want to use the context for some purposes in the future, although there's no context currently for `reconcileNetwork` func. But I check this func again, I find there is no any log or error can be added later, so I agree with you all. I will remove it. Thanks!